### ISSUE TITLE
Indicate how a dataset or model is shared

### ISSUE THREAD

**Issue by tomalrussell** [dafnifacility/user-feedback#37]
It would be useful to add an indication of how the dataset is shared on the dataset detail page.

Currently it shows "Accessed as the owner of the asset".

Suggest including something like "This dataset is shared publicly/(view only)/with members of group X/privately".

**Comment by fmcclean** [comment]
I agree, this info could also be shown on the data catalogue page where all datasets are listed - a simple 'shared' or 'private' would be enough here.

**Comment by fmcclean** [comment]
Also, the option to make a dataset public or share it with a group from the start would be nice. This could be an option in the metadata form.

**Comment by BethanPerkins** [comment]
In the early wireframes for the dataset detail page, we did put in a side-bar box showing who had access to the dataset, but we decided not to implement because - until we can get those links in between people's names and uuids - then it would just show a list of uuids which is likely to be more confusing than helpful.

The groups page will go a long way to simplifying the transparency of who has access to which data. But, yes, more clarity on sharing - especially from the point of view of a data owner - across the platform in general would clearly improve the experience of managing your data on DAFNI. Thanks

Captured in DAFNI internal Ticket [CORE-1460](https://jira.dafni.rl.ac.uk/browse/CORE-1460)
